lib/assets/javascripts/turbolinks.js.coffee

```diff
@@ -203,8 +202,19 @@ onNodeRemoved = (node) ->
     jQuery(node).remove()
   triggerEvent(EVENTS.AFTER_REMOVE, node)
 
-executeScriptTags = (selector) ->
-  scripts = document.body.querySelectorAll(selector)
+getScriptsToRun = (runScripts) ->
+  selector = if runScripts is false then 'script[data-turbolinks-eval="always"]' else 'script:not([data-turbolinks-eval="false"])'
```

Comment:
I think scripts with `data-turbolinks-eval="always"` should always be eval-ed, even when they are in permanent nodes.